docs/content/querying/lookups.md

```diff
@@ -27,6 +27,13 @@ and such data belongs in the raw denormalized data for use in Druid.
 Very small lookups (count of keys on the order of a few dozen to a few hundred) can be passed at query time as a "map" 
 lookup as per [dimension specs](../querying/dimensionspecs.html).
 
+## Configuration
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.lookup.snapshotWorkingDir`| Working path used to store snapshot of current lookup configuration, leaving this property null will disable snapshot/bootstrap utility|null|
```

Comment:
It's possible to use segments without the local segment cache as well, but we force that because it allows the node to be restarted in the exact same state it was in before it went down.  This should be the same.

Actually, though, I think it would make sense to maybe make it default to the segmentCache directory + "/lookups" or something like that.